src/Maestro/tests/Maestro.ScenarioTests/TestParameters.cs

```diff
@@ -21,7 +21,7 @@ public static async Task<TestParameters> GetAsync()
                 .AddUserSecrets<TestParameters>()
                 .Build();
 
-            string maestroBaseUri = Environment.GetEnvironmentVariable("MAESTRO_BASEURI") ?? "https://maestro-int.westus2.cloudapp.azure.com";
+            string maestroBaseUri = Environment.GetEnvironmentVariable("MAESTRO_BASEURI") ??  userSecrets["MAESTRO_BASEURI"] ?? "https://maestro-int.westus2.cloudapp.azure.com";
```

Comment:
The BaseURI isn't really a secret. Why are we reading it out of the userSecrets dictionary?

Reply:
To avoid setting the BaseURI as a env variable every time you re-run ngrok. It's much more easier to change the userSecrets but I'm not the one that actually did that change so Roman might have a better answer to this.

Reply:
Tu run local scenarios you have to run ngrok which generates random  url for BaseURI. In the former code you would have to set env variables and consequently restart visual studio to pick up env changes.
Editing secret json file is better experience, you can do it from visual studio and the changes are applied without restart of visual studio.
Also actually, ngrok URI is kinda secret as attacker could use it to attack your local running services. Luckily ngrok session has limited runtime, half a day max, so the attack window is small and actual risk negligable. But still :-)